tests/ibc-hooks/testutils/contracts/counter/src/msg.rs

```diff
@@ -34,13 +34,30 @@ pub struct GetTotalFundsResponse {
 }
 
 #[cw_serde]
-pub enum SudoMsg {
-    ReceiveAck {
+#[serde(rename = "ibc_lifecycle_complete")]
+pub enum IBCLifecycleComplete {
```

Comment:
LGTM, is the plan to eventually move this into some other crate that can be imported?

Reply:
yeah, I think the outpost repo will have `packages/` for this and other reusable structs that we can publish. Thought not sure if it should be in osmosis-std instead